boards/riscv/longan_nano/support/openocd.cfg

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2021 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+adapter speed 2000
```

Comment:
note: Zephyr openocd seems to bundle gd32vf103.cfg script, isn't that one valid?

Reply:
gd32vf103's nor flash driver is not merged yet.
So we need to use patched OpenOCD at this time.
https://review.openocd.org/c/openocd/+/5839